service/routes.py

```diff
@@ -163,6 +163,74 @@ def delete_wishlists(wishlist_id):
     return make_response("", status.HTTP_204_NO_CONTENT)
 
 
+######################################################################
+# ADD AN ITEM TO WISHLIST
+######################################################################
+@app.route("/wishlists/<string:wishlist_id>/item=<int:item_id>", methods=["PUT"])
```

Comment:
i think method here should be POST

Reply:
![image](https://user-images.githubusercontent.com/98918055/156964637-8c0e95c6-c612-42ea-af55-6624b84c0f1f.png)

Reply:
Thanks for your suggestion! The picture is a comment of issue #3. When an item is added to a wishlist, the wishlist must exist. Therefore, I think adding an item is just updating the items of that wishlist.